<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PickImagesPage</class>
 <widget class="QWidget" name="PickImagesPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="selectImagesButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>330</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>IVE SELECTED ALL OF MY IMAGES</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rejectImagesButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>330</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>ID LIKE TO REVISE MY VISION</string>
   </property>
  </widget>
  <widget class="QPushButton" name="retakePhotoButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>330</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>ID LIKE TO REVISE MY VISION</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
